.circleci/config.yml

```diff
@@ -92,6 +92,9 @@ commands:
      - run:
          name: Check formatting
          command: cd integration_tests && npm run check-fmt
+     - run:
+         name: Check eslint
+         command: cd integration_tests && npx eslint .
```

Comment:
any reason to not add eslint to package.json with a script there?

Reply:
lol, because my js is not great... that is the reason